oeps/oep-0001.rst

```diff
@@ -284,6 +291,12 @@ ReStructuredText [8] allows for rich markup that is relatively easy to read,
 and can also be rendered into good-looking and functional HTML. OEPs are
 rendered to HTML using Sphinx. An `OEP template`_ can be found in the repo.
 
+OEPs may be discussed in a more convenient format, such as a Google Doc, if
+it is deemed most appropriate for the audience. The final OEP should be
+committed as described above with a link to the location of any discussion,
```

Comment:
So even if the discussion happens elsewhere it is expected to be public.  Can we make that more clear here.